Cubical/HITs/Truncation/Base.agda

```diff
@@ -0,0 +1,13 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.HITs.Truncation.Base where
+
+open import Cubical.Data.NatMinusOne
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.HITs.Sn
+
+data  ∥_∥_ {ℓ} (A : Type ℓ) (n : ℕ₋₁) : Type ℓ where
+  ∣_∣ : A  → ∥ A ∥ n
+  top : (f : S (1+ n) → ∥ A ∥ n) → ∥ A ∥ n
+  rays : (f : S (1+ n) → ∥ A ∥ n) (x : S (1+ n)) → top f ≡ f x
```

Comment:
To be consistent with the HoTT book's terminology, in #214 I named these constructors `hub` and `spoke` (as opposed to `top` and `rays`). Is that a change worth making here as well?

Reply:
Sounds like a good idea to me.